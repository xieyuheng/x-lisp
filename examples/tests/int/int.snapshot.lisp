1
0
-1
(iadd 1)
