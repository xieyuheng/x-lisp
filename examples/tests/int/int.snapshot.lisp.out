1
0
-1
(the (-> int? int?) (iadd 1))
